settings/base.py

```diff
@@ -93,6 +92,12 @@
     'django.template.loaders.filesystem.Loader',
```

Comment:
These are the default values, so TEMPLATE_LOADERS can be removed.

https://docs.djangoproject.com/en/1.11/ref/templates/upgrading/